spec/spec_helper.rb

```diff
@@ -3,23 +3,50 @@
   SimpleCov.start
 end
 
-Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
-Dir[File.join(__dir__, "support/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].sort.each { |f| require f }
+Dir[File.join(__dir__, "support/**/*.rb")].sort.each { |f| require f }
 
 require "manageiq-providers-ibm_cloud"
 
+# Iterate through the SSH keys and replace all values.
+# Being very careful with this one as fixing a data leak would be costly.
+def replace_ssh_keys(response)
+  data = JSON.parse(response.body, :symbolize_names => true)
+  return unless data.key?(:keys)
+
+  keys = {:fingerprint => 'SHA256:xxxxxxx', :public_key => 'RSA: VVVVVV'}
+  data.fetch(:keys).each_with_index do |v, i|
+    v.merge!(keys)
+    v[:name] = "random_key_#{i}"
+  end
+  response.body = data.to_json.force_encoding('ASCII-8BIT')
+end
+
+# Replace the contents of the token before writing to file.
+def replace_token_contents(response)
+  data = JSON.parse(response.body, :symbolize_names => true)
+  data.merge!({:refresh_token => 'REFRESH_TOKEN', :ims_user_id => '22222', :expiration => Date.new(2100, 1, 1).to_time.to_i})
+  response.body = data.to_json.force_encoding('ASCII-8BIT')
+end
+
+# Sanitize VPC VCR files.
+def vpc_sanitizier(interaction)
+  # Mask bearer token in recorded file.
+  interaction.request.headers['Authorization'] = 'Bearer xxxxxx' if interaction.request.headers.key?('Authorization')
```

Comment:
re: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/137/files#r577950732  Looks like this accidentally flattens the Array that was there previously, so I'm not sure if that could cause problems or not.

Reply:
Spec tests pass, which is the reason for the VCR file.